containers/podman-desktop
# chore: moving color definitions to dedicated file

### What does this PR do?

We currently add all colors definition to the `color-registry.ts` file. This PR moves them to a dedicated **dependency-free** file.

The last part is important, as for https://github.com/containers/podman-desktop/issues/7439 we need to create a simple CLI tool, that will generate a stylesheet based on those colors, however the ColorRegistry class has too much dependency and `tsx` cannot execute it, as it uses internal imports etc.

### Screenshot / video of UI



### What issues does this PR fix or reference?

Related to https://github.com/containers/podman-desktop/issues/7439

### How to test this PR?

To ensure we did not lost any color while moving them, I run in debugger mode Podman-Desktop, and break on 

https://github.com/containers/podman-desktop/blob/2fa6e77963a7b93de09010117f9b6e04c6e1bbf3/packages/main/src/plugin/color-registry.ts#L219

With my PR checkout I copy the result of `this.listColors('dark')` and `this.listColors('light')` to a dedicated file. Then repeat the operation on the main branch.

Finally I compare using a diff the results from my checkout and the main to ensure no colors has been lost.
